database-plugins/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <artifactId>hydrator-plugins</artifactId>
     <groupId>io.cdap.plugin</groupId>
-    <version>2.10.0</version>
+    <version>2.10.2-SNAPSHOT</version>
```

Comment:
This is the only version that should have changed. No other artifact was released.